src/BODSDataExtractor/extractor.py

```diff
@@ -82,8 +82,9 @@ def check_api_response(self, response, apiResponse):
      
             if response.get("results")==[]:
                 
-                response={'status_code': 404, 'reason': '{"Empty Dataset"}'}
-
+                response={'status_code': 404, 'reason': '{"Empty Dataset, due to invalid "NOC" or "status" entered"}'}

```

Comment:
does this only apply to invalid NOC and status or could it be any parameter?

Reply:
This applies only two these two parameters, the others will have different error messages